farcaster-social-graph-api/farcaster_social_graph_api/main.py

```diff
@@ -17,47 +17,80 @@
+from farcaster_sybil_detection.features.registry import FeatureRegistry
+from farcaster_sybil_detection.features.extractors.network_analysis_extractor import (
+    NetworkAnalysisExtractor,
+)
+from farcaster_sybil_detection.features.extractors.temporal_behavior_extractor import (
+    TemporalBehaviorExtractor,
+)
+from farcaster_sybil_detection.features.extractors.user_identity_extractor import (
+    UserIdentityExtractor,
+)
+
+pl.Config.set_streaming_chunk_size(1_000_000)
+pl.Config.set_fmt_str_lengths(50)
+
+detector_config = Config(
+    data_path=Path(f"{config.DOWNLOAD_DATA_PATH}/../../../data/raw"),
+    checkpoint_dir=Path(f"{config.DOWNLOAD_DATA_PATH}/../../../notebooks/checkpoints"),
+    model_dir=Path(f"{config.DOWNLOAD_DATA_PATH}/../../../notebooks/models"),
+)
+
+registry = FeatureRegistry()
+
+registry.register("user_identity", UserIdentityExtractor)
+registry.register("network_analysis", NetworkAnalysisExtractor)
+registry.register("temporal_behavior", TemporalBehaviorExtractor)
+#registry.register("content_engagement", ContentEngagementExtractor)
+# registry.register("reputation_meta", ReputationMetaExtractor)
+
+detector = DetectorService(detector_config, registry)
 
 app = FastAPI(title="Optimism Farcaster Social Graph API", version="0.1.0")
 
-honeybadger.configure(api_key=config.HONEYBADGER_API_KEY, enviroment=config.ENVIRONMENT)
-app.add_middleware(contrib.ASGIHoneybadger)
+# honeybadger.configure(api_key=config.HONEYBADGER_API_KEY, enviroment=config.ENVIRONMENT)
+# app.add_middleware(contrib.ASGIHoneybadger)
 scheduler = AsyncIOScheduler()
 
 
 @app.on_event("startup")
 async def startup_event():
     """Schedule the cron jobs and start sync_lbo_data when the app starts."""
     try:
-        asyncio.create_task(sync_lbp_data())
-        asyncio.create_task(delete_old_files())
-
-        scheduler.add_job(
-            sync_lbp_data,
-            CronTrigger(day_of_week="tue", hour="0", minute="0"),
-            id="sync_lbp_data",
-            replace_existing=True,
-        )
-
-        scheduler.add_job(
-            delete_old_files,
-            CronTrigger(hour="0", minute="0"),
-            id="delete_old_files",
-            replace_existing=True,
-        )
-
-        logging.info(f"Scheduler started. Jobs: {scheduler.get_jobs()}")
-        scheduler.start()
+        # asyncio.create_task(sync_lbp_data())
+        # asyncio.create_task(delete_old_files())
+
+        # scheduler.add_job(
+        #     sync_lbp_data,
+        #     CronTrigger(day_of_week="tue", hour="0", minute="0"),
+        #     id="sync_lbp_data",
+        #     replace_existing=True,
+        # )
+
+        # scheduler.add_job(
+        #     delete_old_files,
+        #     CronTrigger(hour="0", minute="0"),
+        #     id="delete_old_files",
+        #     replace_existing=True,
+        # )
+
+        # logging.info(f"Scheduler started. Jobs: {scheduler.get_jobs()}")
+        # scheduler.start()
```

Comment:
would confirm this with @devjoaov

Reply:
Actually just forgot to uncomment this. I commented because starting up on every test was taking too long, I think I will add a checkpoint there